solidity/contracts/BurntPixArchives.sol

```diff
@@ -51,6 +52,7 @@ contract BurntPixArchives is LSP8CappedSupply {
     bytes32 public immutable burntPicId;
     uint256 public immutable winnerIters;
     uint256 public archiveCount;
+    uint256 public currentHighestLevel = 0;
```

Comment:
what is this used for?

Reply:
@Ledespana used for tracking what's the highest fibonacci level a user has hit so that we alter opacity of the border based on that